app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -792,6 +792,43 @@ def save(self):
             return experiment
 
 
+class NimbusExperimentCSVSerializer(
+    NimbusExperimentBranchMixin,
+    NimbusStatusValidationMixin,
+    NimbusExperimentDocumentationLinkMixin,
+    ExperimentNameValidatorMixin,
+    serializers.ModelSerializer,
+):
+    experiment_name = serializers.CharField(source="name")
+
+    product_area = serializers.CharField(source="application")
+
+    rollout = serializers.BooleanField(source="is_rollout")
+    owner = serializers.SlugRelatedField(read_only=True, slug_field="email")
```

Comment:
Do we need `read_only` here? is this because it's a slug field?

Reply:
since - Relational field must provide a `queryset` argument, override `get_queryset`, or set read_only=`True`.